starlette_admin/contrib/sqla/helpers.py

```diff
@@ -23,8 +43,8 @@
     "not_contains": lambda f, v: not_(cast(f, String).contains(v)),
     "is_false": lambda f, v: f == false(),
     "is_true": lambda f, v: f == true(),
-    "is_null": lambda f, v: f.is_(None),
-    "is_not_null": lambda f, v: f.is_not(None),
+    "is_null": lambda f, v: __is_null(f),
+    "is_not_null": lambda f, v: __is_not_null(f),
```

Comment:
I wasn't sure if there is a specific coding style to follow. If there is, please make any necessary modifications with pleasure